### PR TITLE
Create module

### PR DESCRIPTION
**Pull request by jgomezve**

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -8,24 +8,188 @@ variable "name" {
...
+  validation {
+    condition     = var.port >= 0 && var.port <= 127
+    error_message = "Minimum value: `1`. Maximum value: `127`."
+  }
+}
+
+variable "sub_port" {
+  description = "Access SPAN Destination Groups Sub-port. Minimum value: `1`. Maximum value: `16`."
+  type        = number
+  default     = 0
```
Comment (jgomezve) [inline variables.tf:48]: @danischm 
The optional variables of type number are set with a default value of `0`. The validation range is adapted accordingly. Do you agree with this approach?

File: main.tf [hunk 2]
```diff
@@ -1,9 +1,64 @@
...
+  dn         = "${aci_rest_managed.spanDest.id}/rsdestPathEp-[${format("topology/pod-%s/paths-%s/pathep-[%s]", var.pod_id, var.node_id, var.channel)}]"
+  class_name = "spanRsDestPathEp"
+  content = {
+    mtu = var.mtu
+    tDn = format("topology/pod-%s/paths-%s/pathep-[%s]", var.pod_id, var.node_id, var.channel)
+  }
+}
+
+resource "aci_rest_managed" "spanRsDestEpg" {
+  count      = var.tenant != "" && var.application_profile != "" && var.endpoint_group != "" && var.ip != "" && var.source_prefix != "" && var.channel == "" && var.node_id == 0 ? 1 : 0
```
Comment (jgomezve) [inline main.tf:50]: @danischm I created this "complex" logic to avid the creation of a `spanRsDestEpg` and ` `spanRsDestPathEp` resource. The logic could also be handled in the nac modules. What do you think about this ?

Comment (danischm) [reply main.tf:50]: I would keep it here.

File: variables.tf (hunk 1 above)
Comment (danischm) [reply variables.tf:48]: Yes, that should be fine.

**Review by danischm — APPROVED** [review]

**Pull request merged**